lib/testing/src/lib/core/pages/user-info-dialog.page.ts

```diff
@@ -15,11 +15,11 @@
  * limitations under the License.
  */
 
-import { Util } from '../../../util/util';
-import { TabsPage } from '../material/tabsPage';
 import { element, by, browser, protractor } from 'protractor';
+import { BrowserVisibility } from '../browser-visibility';
+import { TabsPage } from '../../material/tabsPage';
```

Comment:
wrong path

Reply:
this file is in the adf-testing as well, hence using a relative path.